README.md

```diff
@@ -1,191 +1,264 @@
 # @eik/node-client
 
-The Eik Node.js client facilitates loading Eik config and providing URLs to assets on an Eik server or in local development plus loading import maps from the Eik server.
+This is an Eik utility for servers running on Node. With it you can:
```

Comment:
```suggestion
This is an Eik utility for working with [@eikservice](https://github.com/eik-lib/service) and your configuration. With it you can:
```
This feels a bit vague, as there is no other option than to run an Eik server on node using the `service` module?